packages/lib/event-types/transformers/api-request.ts

```diff
@@ -15,6 +16,7 @@ import {
   BookingWindowPeriodInputTypeEnum_2024_06_14,
   BookingWindowPeriodOutputTypeEnum_2024_06_14,
 } from "@calcom/platform-types/event-types/event-types_2024_06_14/inputs/enums/booking-window.enum";
+import { Frequency } from "@calcom/platform-types/event-types/event-types_2024_06_14/inputs/enums/frequency";
```

Comment:
same here, import from "@calcom/platform-types" after exporting to index.js

Reply:
Hi @ThyMinimalDev 
if we export the file inside of platform/types/index.js and import from "@calcom/platform-types" the build fails, 
not sure why.

Reply:
What if the enum is exported from `packages/platform/types/event-types/event-types_2024_06_14/inputs/enums/index.ts` and then imported as suggested?

Reply:
I made the suggested changes. in https://github.com/calcom/cal.com/pull/16251/commits/464d2611ed19dbd6ae390332b4d6dd5884bcae3b  However, I’m encountering an issue where the platform libraries build is failing.